hadoop-ozone/interface-client/src/main/proto/OMAdminProtocol.proto

```diff
@@ -69,16 +69,58 @@ message DecommissionOMResponse {
     optional string errorMsg = 3;
 }
 
+/** Asks OM to reload configuration file. */
+message StartOmReconfigurationRequestProto {
+}
+
+message StartOmReconfigurationResponseProto {
+}
+
+/** Query the running status of reconfiguration process */
+message GetOmReconfigurationStatusRequestProto {
+}
+
+message GetOmReconfigurationStatusConfigChangeProto {
+  required string name = 1;
+  required string oldValue = 2;
+  optional string newValue = 3;
+  optional string errorMessage = 4;  // It is empty if success.
+}
+
+message GetOmReconfigurationStatusResponseProto {
+  required int64 startTime = 1;
+  optional int64 endTime = 2;
+  repeated GetOmReconfigurationStatusConfigChangeProto changes = 3;
+}
+
+/** Query the reconfigurable properties on OM. */
+message ListOmReconfigurablePropertiesRequestProto {
+}
+
+message ListOmReconfigurablePropertiesResponseProto {
+  repeated string name = 1;
+}
```

Comment:
I think these messages should be defined in a service-agnostic way (in `hdds.proto` and without references to `Om`) to avoid duplication if we would like to implement similar feature in SCM and other services.